content/program/talks/contents.lr

```diff
@@ -0,0 +1,5 @@
+_model: talks
+---
+title: Talks
+---
+show_in_menu: yes
```

Comment:
Would it be possible to have this below the schedule? Seems like it might make the most sense:
```suggestion
show_in_menu: yes
---
sort_index: 1
```